Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -542,7 +542,7 @@
             return;
 
         if (state.commitLayerTreeMessageState == NeedsDisplayDidRefresh)
-            didRefreshDisplay(process.connection());
+            didRefreshDisplay(process.protectedConnection().get());
```

Comment:
You failed to protect the connection before calling `waitForAndDispatchImmediately()` on it a few lines above.

Reply:
Will fix.